Doc/installing/index.rst

```diff
@@ -220,14 +220,14 @@ There are also additional resources for `installing pip.
 Installing binary extensions
 ----------------------------
 
-Python has typically relied heavily on source based distribution, with end
-users being expected to compile extension modules from source as part of
+Python has typically relied heavily on source based distributions, with end
+users being expected to compile binary extensions from source as part of
```

Comment:
_extension modules_ is the way Python calls these (native code shared libraries built from other languages)